src/components/services/Plex.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <Generic :item="item">
+    <template #indicator>
+      <div class="notifs">
+        <strong v-if="streams > 0" class="notif activity" title="Active Streams">
+          {{ streams }}
+        </strong>
+        <strong v-if="series > 0" class="notif series" title="Total Series">
+          {{ series }}
+        </strong>
+        <strong v-if="movies > 0" class="notif movies" title="Total Movies">
+          {{ movies }}
+        </strong>
+        <strong v-if="warnings > 0" class="notif warnings" title="Warning">
+          {{ warnings }}
+        </strong>
+        <strong v-if="errors > 0" class="notif errors" title="Error">
+          {{ errors }}
+        </strong>
+        <strong v-if="serverError" class="notif errors" title="Connection error to Plex API, check url and token in config.yml">
+          ?
+        </strong>
+      </div>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+export default {
+  name: "Plex",
+  components: {
+    Generic,
+  },
+  mixins: [service],
+  props: {
+    item: Object,
+  },
+  data: () => {
+    return {
+      streams: null,
+      series: null,
+      movies: null,
+      warnings: null,
+      errors: null,
+      serverError: false,
+    };
+  },
+  created: function () {
+    const checkInterval = parseInt(this.item.checkInterval, 10) || 0;
+    if (checkInterval > 0) {
+      setInterval(() => this.fetchData(), checkInterval);
+    }
+    this.fetchData();
+  },
+  methods: {
+    fetchData: function () {
+      const handleError = (e) => {
+        console.error(e);
+        this.serverError = true;
+      };
+      
+      fetch(`${this.endpoint}/status/sessions?X-Plex-Token=${this.item.token}`)
+        .then((response) => response.text())
```

Comment:
Same here about using this.fetch instead. To get the text response, set the last arg to false
```suggestion
      this.fetch(`/status/sessions?X-Plex-Token=${this.item.token}`, {}, false)
```